t/validator_matches_date_format.t

```diff
@@ -2,35 +2,46 @@
 
 use strict;
 use lib qw(../lib lib ../t t);
-use TestUtil;
 use Test::Exception;
 use DateTime;
-use Test::More tests => 6;
+use Test::More;
 
-require_ok( 'Workflow::Validator::MatchesDateFormat' );
+require_ok( 'Workflow::Validator::MatchesDateFormat');
 
 my $validator;
 my $wf;
 
-dies_ok { $validator = Workflow::Validator::MatchesDateFormat->new({}) };
+dies_ok { $validator = Workflow::Validator::MatchesDateFormat->new({}) } 'Constructor without parameters, we die';
+
+dies_ok { $validator = Workflow::Validator::MatchesDateFormat->new({ date_format => bless {} }) } 'Constructor with bad parameters, we die';
 
 ok($validator = Workflow::Validator::MatchesDateFormat->new({
```

Comment:
Isn't this a 'lives'?

Reply:
Hmmm it throws a configuration exception if the `date_format` parameter is not supplied to the constructor in Workflow::Validator::MatchesDateFormat

REF: https://github.com/jonasbn/perl-workflow/blob/2857db36d3dcb9204e23dfddbd22855a695d52fa/lib/Workflow/Validator/MatchesDateFormat.pm#L17-L20